Maquetado/scss/footer.scss

```diff
@@ -0,0 +1,28 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  height: $foot-height;
+  justify-content: center;
+  left: 0;
+  position: absolute;
+  width: 100%;
+
+  .foot-slogan {
+    color: $white;
+    font-size: $text-big;
+    margin-bottom: 40px;
+  }
+
+  .foot-logo {
+    height: 70px;    
+  }  
+
+  .foot-rights {
+    text-align: right;
+    color: $white;
+    font-size: $text-small;
+    width: 95%;
```

Comment:
I think there's no need to set this width, let it adapt automatically

Reply:
we also try to avoid uncommon percentage numbers since they vary according to every device width and that might generate some problems.